docs/Animations.md

```diff
@@ -444,14 +445,14 @@ var App = React.createClass({
 
     return (
       <View style={styles.container}>
-        <TouchableWithoutFeedback onPressIn={this._onPressIn}
-                                  onPressOut={this._onPressOut}>
+        <TouchableWithoutFeedback onPressIn={() => this._onPressIn()}
+                                  onPressOut={() => this._onPressOut()}>
```

Comment:
@nicklockwood Yeah, no auto-binds in ES6 classes, unless you use class properties.